modules/Project/chart.php

```diff
@@ -628,47 +653,47 @@ public function year_week($start_date, $end_date)
 		foreach ( $period as $dt )
 		{
 			$count++;
-			$aResult[$dt->format('Y')][$dt->format('M')][] = $dt->format('W');
-		}
-
-		return array($aResult, $count);
-	}
+      $aResult[$dt->format('Y')][$GLOBALS['app_list_strings']['dom_cal_month_short'][$dt->format('n')]][] = $dt->format('W');
```

Comment:
Line is longer than 120 characters and it is also difficult to read.Consider assigning the $GLOBALS and $dt->format methods into variables and then use the variables instead.

```
public function <name>
{
    global $app_list_strings;
    // ... 
    foreach ($period as $dt) {
        // Perhaps give better names here
        $n = $GLOBALS['app_list_strings']['dom_cal_month_short'][$dt->format('n')];
        $aResult[$dt->format('Y')][$n][] = $dt->format('W');
    }
    // ...
}

```